bigquerydatatransfer/quickstart.rb

```diff
@@ -12,29 +12,32 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-# [START bigquerydatatransfer_quickstart]
-# Imports the Google Cloud client library
-require "google/cloud/bigquery/data_transfer"
+def quickstart project_id:
+  # [START bigquerydatatransfer_quickstart]
+  # [START require_library]
+  # Imports the Google Cloud client library
+  require "google/cloud/bigquery/data_transfer"
+  # [END require_library]
 
-# Your Google Cloud Platform project ID
-project_id = "YOUR_PROJECT_ID"
+  # Your Google Cloud Platform project ID
+  # project_id = "YOUR_PROJECT_ID"
```

Comment:
@TheRoyalTnetennba I wasn't sure what to do about this line in the sample. Just remove it, as you did in #542?

Reply:
yeah. might have to wrap the sample in a method so it's easier to test.

Reply:
I'm not sure I understand. What change should I make here?

Reply:
OK, based on similar [argument "initialization comments" in 542](https://github.com/GoogleCloudPlatform/ruby-docs-samples/blob/0b1fa9a0f3541eb32ec2a88b101db75d935c488c/logging/quickstart.rb#L27-L30), this should be good as-is. Thanks.